numba_cuda/numba/cuda/tests/cudadrv/test_linker.py

```diff
@@ -329,6 +329,20 @@ def test_get_local_mem_per_specialized(self):
         calc_size = np.dtype(np.float64).itemsize * LMEM_SIZE
         self.assertGreaterEqual(local_mem_size, calc_size)
 
+    def test_link_for_different_cc(self):
+        if not config.CUDA_USE_NVIDIA_BINDING:
+            self.skipTest("Ctypes Linker does not support outputting PTX.")
+
+        linker = _Linker.new(cc=(12, 0))
+        code = """
+__device__ int foo(int x) {
+    return x + 1;
+}
+"""
+        linker.add_cu(code, "foo")
+        ptx = linker.complete()
```

Comment:
We can get the PTX from the linker like:

```suggestion
        ptx = linker.get_linked_ptx().decode()
```